js/README.md

```diff
@@ -203,6 +197,28 @@ The JS community is a diverse group with a varied list of target environments an
 
 If you think we missed a compilation target and it's a blocker for adoption, please open an issue.
 
+# Community
+
+Full list of broader Apache Arrow [committers](https://arrow.apache.org/committers/) and [projects & organizations](https://github.com/apache/arrow/blob/master/site/powered_by.md).
+
+## Committers
+
+* Brian Hulette, CCRI
```

Comment:
Technically I'm a contributor and not a committer, not sure how important it is to make that distinction